newinstaller/style.css

```diff
@@ -0,0 +1,221 @@
+}
+
+.header-1 {
+    font-family: var(--header-font);
+    color: var(--text-light);
+    font-size: var(--h1-fontsize);
+    margin: 0;
+    padding: 1rem;
+}
+
+.breadcrumbs {
+    font-family: var(--header-font);
+    font-size: var(--text-large);
+    display: flex;
+    align-items: baseline;
+    list-style: none;
+    padding-left: .15rem;
+    margin-bottom: .5rem;
+}
+
+.arrow_icon {
+    margin-left: 0.5rem;
+    margin-right: 0.5rem;
+    font-size: var(--text-large);
+}
+
+.header-2 {
+    font-family: var(--header-font);
+    color: var(--primary-color);
+    font-size: var(--h2-fontsize);
+    margin-top: 0rem;
+    margin-bottom: 1.5rem;
+}
+
+/*  Container for the main content of the installer
+    window, including the .body-text and .input-grid    */
+.inner-wrapper {
+    padding: 2rem;
+    padding-top: 0;
+}
+
+.body-text {
+    font-family: var(--body-font);
+    margin-bottom: 2.5rem;
+}
+
+/* Group of all input fields + labels*/
+.input-grid {
+    font-family: var(--body-font);
+    display: grid;
+    grid-template-columns: 17rem 17rem;
+    grid-template-rows: repeat(auto, 1fr);
+    grid-column-gap: 2rem;
+    grid-row-gap: 1rem;
+}
+
+/*  Media query for changing element disposition to support
+    smaller screens like tablets and smaller browser windows */
+@media (max-width: 768px) {
+
+    .input-grid{
+        font-family: var(--body-font);
+        display: flex;
+        flex-direction: column;
+        gap: 1.5rem;
+    }
+
+    .header h1 {
+        font-family: var(--header-font);
+        color: var(--text-light);
+        font-size: var(--h1-fontsize-small);
+        margin: 0;
+        padding: 1rem;
+    }
+
+    h2 {
+        font-family: var(--header-font);
+        color: var(--primary-color);
+        font-size: var(--h2-fontsize-small);
+        margin-top: 0rem;
+        margin-bottom: 1.5rem;
+    }
+}
+
+/*  Lets the last cell of the .input-grid span 2 columns */
+.grid-element-span {
+    grid-area: 4 / 1 / 5 / 3;
+}
+
+/*  Sets label above corresponding input field  */
+.input-field {
+    display: flex;
+    flex-direction: column;
+}
+
+input[type=text] {
+    inset: none;
+    border-radius: 0;
+    border: 1px solid var(--text-dark);
+    padding: 0.25rem;
+    width: 12rem;
+    outline: none;
+    transition: .3s ease;
+}
+
+input[type=text]:focus {
+    border-color: var(--primary-color);
+}
+
+.tip {
+    font-size: var(--text-xs);
+    font-style: italic;
+    color: var(--text-dark);
+    margin-top: .25rem;
+}
+
+.buttonContainer {
+    display: flex;
+    justify-content: space-between;
+}
+
+.checkbox {
+    display: flex;
+    font-size: var(--text-medium);
+}
+
+input[type=checkbox] {
+    accent-color: var(--primary-color);
+}
+
+.warning {
+    font-size: var(--text-xs);
+    color: var(--warning-text);
+    font-style: italic;
+    padding-left: 0.25rem;
+    margin-top: 0.25rem;
+}
+
+.backButton {
+    background-color: rgba(0, 0, 0, 0); /* To get none color*/
```

Comment:
remove comment, declarations that should be obvious doesn't need to be commented